2.overview_and_terminology.md

```diff
@@ -176,7 +189,7 @@ Annotations provide a mechanism for attaching arbitrary text within the metadata
 
 > Annotations are key/value pairs. Valid annotation keys have two segments: an optional prefix and name, separated by a slash (/). The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between. The prefix is optional. If specified, the prefix must be a DNS subdomain: a series of DNS labels separated by dots (.), not longer than 253 characters in total, followed by a slash (/).
 
-The following annotation labels are _predefined_ and are _strongly recommended_.
+The following annotation labels are _predefined_ and NOT RECOMMENDED.
```

Comment:
why is this not recommended with an example using it right below?

Reply:
My bad. The “NOT” is not supposed to be there.

Reply:
Fixed